packages/react-hooks/src/hooks/media.ts

```diff
@@ -3,8 +3,16 @@ import {useState, useEffect, useLayoutEffect} from 'react';
 type EffectHook = typeof useEffect | typeof useLayoutEffect;
 
 function createUseMediaFactory(useEffectHook: EffectHook) {
-  return (query: string) => {
-    const [match, setMatch] = useState(false);
+  return (
+    query: string,
+    {ssr, ssrValue}: {ssr?: boolean; ssrValue?: boolean} = {
+      ssr: false,
+      ssrValue: false,
+    },
+  ) => {
+    const [match, setMatch] = useState(() =>
+      ssr ? Boolean(ssrValue) : window.matchMedia(query).matches,
```

Comment:
Should we just do this?
```suggestion
      ssrValue !== undefined ? Boolean(ssrValue) : window.matchMedia(query).matches,
```